eisbuk-admin/src/components/atoms/DateSwitcher/DateSwitcher.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { DateTime, DateTimeUnit } from "luxon";
+import { useDispatch } from "react-redux";
+
+import { CalendarPicker } from "@mui/lab";
+
+import { changeCalendarDate } from "@/store/actions/appActions";
+import Menu from "@mui/material/Menu";
```

Comment:
Here you can import `MenuProps` (as a named import)...Mui luckily provides the props interface for each of their components right by the component file...like we do in each `<componentFolder>/index.ts`